src/testsuite/arith128_test_i128.c

```diff
@@ -2308,7 +2308,8 @@ test_msumudm (void)
 int
 test_muludq (void)
 {
-  vui32_t i, j, k, l /*, m*/;
+  vui32_t i, j, k/*, l , m*/;
```

Comment:
Do you want to keep the obsolete code in the comment?

Reply:
Leave it for now. Fix it in upstream.